assets/stylesheets/responsive/custom.scss

```diff
@@ -458,21 +458,16 @@ a.link_button_green_large {
 
 div.correspondence {
   background-color: $correspondence-bg;
-  padding-bottom: 0.5em;
-  .event_actions {
-    margin-bottom: 0;
```

Comment:
(It discards the use of Courier for incoming messages and - incompletely - allows the new link to this layout to work)